### PR TITLE
ATO-79: 404 error page

### PR DESCRIPTION
**Pull request by hannah-leong**
## What?

Add a 404 error page for the orchestration frontend

## Why?

When a user attempts to access a path that does not exist then we should display a Page not found page. 

## Change have been demonstrated

The Page not found page is displayed when a user tries to access a path that does not exist on the Orchestration frontend

### CODE REVIEW

File: src/config/nunchucks.ts
Comment (ethanmills) [inline src/config/nunchucks.ts:1]: Why is this called nunchucks not nunjucks? Am I missing something?

**Review by ethanmills — COMMENTED** [review]
Would be good to have a test covering the translations (e.g. if I have a cy cookie I get some Welsh back)

File: src/config/nunchucks.ts
Comment (hannah-leong) [reply src/config/nunchucks.ts:1]: That is how it is in authentication - we also were a bit confused but assumed it was for differentiating them

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,12 @@
...
\ No newline at end of file
+node_modules/
+
+# dotenv environment variables file
+.env
+
+# Additional ignores
+.idea
+
+src/assets/javascript/all.js
```
[the remark this answers is not in the document]
Comment (hannah-leong) [reply .gitignore:13]: Something we took from authentication but I realise we ended up removing the javascript from the assets directory so do not need this. Will remove

File: src/config/nunchucks.ts
Comment (ethanmills) [reply src/config/nunchucks.ts:1]: I'd rather call this `templating` or something than make up a word I think

**Review by ethanmills — APPROVED** [review]

**Pull request merged**